src/main/resources/admin/tools/content-type-manager/components/input/component-input.html

```diff
@@ -81,30 +81,28 @@
       data() {
         return {
           showConfig: false,
-
           map: {},
-
-          commonFieldNames: {
-            occurrences: {
-              type: "element",
-              name: "occurrences",
-              attributes: { minimum: 0, maximum: 1 },
-            },
-            "help-text": {
-              type: "element",
-              name: "help-text",
-              elements: [{ type: "text", text: "" }],
-            },
-            label: {
-              type: "element",
-              name: "label",
-              elements: [{ type: "text", text: "" }],
-            },
-          },
         };
       },
       created() {
         this.map = app.$root._data.map;
+
+        // TODO: Add cache to avoid unecessary requests
+        fetch(
+            `http://localhost:8080/_/service/tool.content.type.manager/test2?schemaName=${this.inputType}`
```

Comment:
Localhost? This will not work if you use the cty-manager outside of a local environment. 
And test2 is not a good name for this service.

Reply:
I created an issue (#73) to update the name of the services since currently the are test1, test2 and test3. 

For now I'll refactor the localhost (my bad :sweat_smile:).